lib/collective.lib.js

```diff
@@ -117,11 +117,11 @@ export const getTopContributors = contributors => {
 };
 
 export const isEmptyCollectiveLocation = account => {
-  if (!account?.location) {
+  if (!account?.location?.name) {
```

Comment:
Most collectives and events have an empty location `name`. Plus, this will return `false` if `name` is `Online` - thus hiding the `Location` section while we want to keep showing it if there are private instructions.